config/routes.rb

```diff
@@ -15,8 +15,8 @@
 
   resources :submissions
 
-  resources :users
+  resource :profile, only: [:edit, :update]
```

Comment:
Nice use of a singular resource.  That way, users can't enter in any other `id` as a URL parameter.